### PR TITLE
add author data for each posts

### PR DESCRIPTION
**Pull request by geekyahmed**
I added a new field in the ```posts.json``` file named ``` author ``` to store the name of the author and I also added the author's name into each post for rendering,

### CODE REVIEW

File: posts/firstpost.md [hunk 1]
```diff
@@ -4,6 +4,7 @@ description: This is a post on My Blog about agile frameworks.
 date: 2018-05-01
 tags:
   - another-tag
+author: John Doe
```
Comment (cramforce) [inline posts/firstpost.md:7]: I'm not clear as to why this is needed.

**Comment by geekyahmed** [comment]
Well, it's for the author name

**Comment by cramforce** [comment]
It's maintained globally in metadata.json. I don't see any consumer for the information you added.

**Comment by indcoder** [comment]
@geekyahmed  But this is not required in the upstream template. Even we have that requirement i.e. each blog post may have different authors who we want to acknowledge [essentially authorship is at the post level; not at the blog level which is what the template is originally developed as] . So while we are customizing this on our fork, but since its not the use case for general use of the template [at least the template is not structured for this use case right now and as for the near future @cramforce  will have to confirm it the direction], we will not be offering this as a PR into the upstream.

**Pull request closed without merging**